app/src/main/java/com/sdp13epfl2021/projmag/activities/ProjectInformationActivity.kt

```diff
@@ -417,6 +426,18 @@ class ProjectInformationActivity : AppCompatActivity() {
         }, { showToast(getString(R.string.could_not_download_video), Toast.LENGTH_LONG) })
     }
 
+    private fun deletingAndCopyingVideo(fileUrl: String, deleteDirectory: File, copyDirectory: File){
```

Comment:
Using names like moveVideoFile(fileUrl: String, fromDirectory: File, toDirectory: File) or other names would be better to understand.